module3/labs/week1/HowTheWebWorks.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: page
+title: How the Web Works Lab
+---
+
+<aside class="instructor-notes">
+    <p><strong>Instructor Note</strong><br>This lab should be an individual or small group exploration.  Before breaking for worktime, show students how to open their Comman Prompt so that they can try out `curl`</p>
+</aside>
+
+## Lab Objectives
+* Reinforce understanding of the HTTP request/response cycle
+* Use a basic client to see the cycle in action
+
+## Requests and Responses in Detail
+## HTTP Requests and Responses
+
+The HyperText Transfer Protocol gives us rules about how messages should be sent around the Internet. The system that initiates a connection sends a "request", and the system the answers sends a "response".
+
+### HTTP Request
+
+When a "client" (like a web browser) retrieves information, it sends a payload of data to a server as a "request". This request is made up of three main parts:
+
+- A Request line, containing three piece of information:
+  - the HTTP **verb** (also called an http method) for sending or retrieving information
+  - the URI **path** of the resource where we're sending or retrieving information
+  - the version of the HTTP protocol our "client" software is using, usually `HTTP/1.1`
+
+- Headers, which are key/value pairs, which contain supplemental information about our request
+
+- An optional body; we only send data to the server in the body when we are creating or modifying something
+
+### HTTP Response
+
+When the server or web application is finished processing our request, it will send back a response which is a payload of data, and is made up of three main parts:
+
+- a Status line, containing three pieces of information:
+  - The version of the HTTP protocol that this response is using
+  - a 3-digit numeric "status code"
+  - a user-friendly string description of what the "status code" means
+
+- Headers, also sent as key/value pairs similar to the HTTP request
+
+- An optional body; almost all responses will contain additional data in the body. In mod 2, our "body" payload will almost always be HTML.
```

Comment:
I think this is lingering from BE. Probably best to remove this line, Mod 3 is a ways away.